content/reference/sql/supported-clients/sql-editors.md

```diff
@@ -7,40 +7,101 @@ title: SQL Editors
 Dolt comes with a built in MySQL compatible server that you can connect SQL Editors too. Here are a list of notable MySQL Editors and our compatibility status.
 | Editor | Supported | Notes and limitations |
 | :--- | :--- | :--- |
-| [Tableplus](https://tableplus.com/) | ✅ | |
+| [Tableplus](https://tableplus.com/) | ✅ | Recommended (see below) |
```

Comment:
If we're going to make a strong recommendation like this, we need to have very strong assurances it actually works. It's my understanding that Dolt still has a lot of issues with TablePlus, e.g. info schema queries that don't work. Is that not the case?

Reply:
It's the most workable. By pushing users to one editor we can streamline most of these changes instead of hopping around between different editors with quick fixes.